src/components/WarningAnimateMessage/index.js

```diff
@@ -85,8 +84,8 @@ const styles = StyleSheet.create({
         elevation: 1,
         borderRadius: 8,
         width: 259,
-        backgroundColor: COLORS.white,
-        borderColor: COLORS.concrete,
+        backgroundColor: 'white',
+        borderColor: '#f2f2f2',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color values 'white' and '#f2f2f2' could be moved to a centralized location for better maintainability. If these colors are used in multiple places, it would be easier to change them from one place rather than searching through the entire codebase.

```diff
-        backgroundColor: 'white',
-        borderColor: '#f2f2f2',
+        backgroundColor: colors.white,
+        borderColor: colors.concrete,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->